src/matrix/e2ee/olm/types.ts

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2022 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+export type OlmMessage = {
+    type?: 0 | 1,
```

Comment:
I think this might be an esbuild limitation. [This](https://vitejs.dev/guide/features.html#typescript-compiler-options) says: 
> It is because esbuild only performs transpilation without type information, it doesn't support certain features like const enum and implicit type-only imports.

Reply:
Aha, thanks